DOCSP-33135 What's New AWS Region Settings Link (#985)
mongodb/docs-node: source/whats-new.txt

  ``Decimal128.fromStringWithRounding()`` method. To learn more, see the
  `v6.1.0 bson release notes <https://github.com/mongodb/js-bson/releases/tag/v6.1.0>`__.

- Detects environment variables for region settings when you

>   authenticate by using `IAM AssumeRoleWithWebIdentity 
>   <https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html>`__ 

Reviewer: ```suggestion
  authenticate by using the `IAM AssumeRoleWithWebIdentity action
  <https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html>`__ 
```

  with ``MONGODB-AWS`` authentication. To instruct the driver to use your region options, 
  you must set both of the following environment variables:

  - ``AWS_STS_REGIONAL_ENDPOINTS``
  - ``AWS_REGION``

>   For more information on how to set region settings for ``MONGODB-AWS`` 
>   authentication, see the :guilabel:`Web Identity Token` tab in the 
>   :ref:`MONGODB-AWS <mongodb-aws>` section of the Authentication Mechanisms guide. 

Reviewer: ```suggestion
  To learn how to set region settings when using the ``MONGODB-AWS`` 
  authentication mechanism, see the :guilabel:`Web Identity Token` tab in the 
  :ref:`MONGODB-AWS <mongodb-aws>` section of the Authentication Mechanisms guide. 
```
